ygot/render.go

```diff
@@ -1354,20 +1355,7 @@ func jsonAnnotationSlice(v reflect.Value) (interface{}, error) {
 
 	vals := []interface{}{}
 	for i := 0; i < v.Len(); i++ {
-		fv := v.Index(i).Interface().(Annotation)
-		jv, err := fv.MarshalJSON()
-		if err != nil {
-			return nil, fmt.Errorf("cannot marshal annotation %v type %T to JSON: %v", fv, fv, err)
-		}
-
-		// MarshalJSON returns []byte, but we really want to have this as the unmarshalled
-		// value, since constructJSON returns a series of map[string]interface{} Which
-		// are later marshalled, we therefore unmarshal the []byte into an interface{}
-		var nv interface{}
-		if err := json.Unmarshal(jv, &nv); err != nil {
-			return nil, fmt.Errorf("annotation %v, type %T could not be unmarshalled from JSON: %v", fv, fv, err)
-		}
-		vals = append(vals, nv)
+		vals = append(vals, deepcopy.Copy(v.Index(i).Interface().(Annotation)))
```

Comment:
Can you add a comment here? I assume that because we have defined that the `Annotation` must be JSON marshallable itself then we don't need to marshal it in this case?

Reply:
Yes that's right, done.